File: 艾诗缇/mdx_gyl/androidapis/com/mdx/store/data/proto/apis/Apikcpd_sp.java
//
//  Apikcpd_sp
//
//  Created by liulu on 2014-09-25 18:52:22
//  Copyright (c) liulu All rights reserved.


/**
   查询库存量 userkeya
*/

package com.mdx.store.data.proto.apis;

import android.content.Context;
import com.mdx.framework.server.api.ApiManager;
import com.mdx.framework.server.api.UpdateOne;


public class Apikcpd_sp extends com.mdx.framework.server.api.ApiUpdate {


	/**
	 * 查询库存量 userkeya
	 * @param context 上下文
	 * @param parent  调用接口实体
	 * @param method  返回到哪个方法
	 * @callback com.mdx.store.data.proto.Gongying.KcpdSpList
	*/
	public UpdateOne get(Context context, Object parent, String method) {
		this.setMethod(method);
		this.setContext(context);
		this.setParent(parent);
		UpdateOne update = new UpdateOne("kcpd_sp", new String[][] { 
		  });
		return initUpdateOne(update);
	}

	/**
	 * 查询库存量 userkeya
	 * @param context 上下文
	 * @param parent  调用接口实体
	 * @param method  返回到哪个方法
	 * @callback com.mdx.store.data.proto.Gongying.KcpdSpList
	*/
	public UpdateOne load(Context context, Object parent, String method) {
		UpdateOne update=get(context, parent, method);
		ApiManager.Load(getContext(), getParent(), new UpdateOne[] { update });
		return update;

	}

	/**
	 * 查询库存量 userkeya
	 * @param context 上下文
	 * @param parent  调用接口实体
	 * @param method  返回到哪个方法
	 * @callback com.mdx.store.data.proto.Gongying.KcpdSpList
	*/
	public Apikcpd_sp set() {
		get(null,null,null);
		return this;
	}


}

File: 艾诗缇/mdx_gyl/iosApiJson/Apikcpd_sp.m
//
//  Apikcpd_sp
//
//  Created by liulu on 2014-09-25 18:52:23
//  Copyright (c) liulu All rights reserved.


/**
   
*/

#import "Apikcpd_sp.h"

@implementation Apikcpd_sp


	/**
	 * 查询库存量 userkeya
	 * @param delegate 回调类
	 * @param select  回调函数
	 * @callback KcpdSpList
	*/
	-(UpdateOne*)get:(id)delegate selecter:(SEL)select  {
		NSMutableArray *array=[[NSMutableArray alloc]initWithObjects:nil];
		UpdateOne *updateone=[[Updateone2Json alloc] init:@"kcpd_sp" params:array  delegate:delegate selecter:select];
		return [self instanceUpdate:updateone];
	}

	/**
	 * 查询库存量 userkeya
	 * @param delegate 回调类
	 * @param select  回调函数
	 * @callback KcpdSpList
	*/
	-(UpdateOne*)load:(id)delegate selecter:(SEL)select  {
		UpdateOne *update=[self get:delegate selecter:select];
		[DataManager loadData:[[NSArray alloc]initWithObjects:update,nil] delegate:delegate];
		return update;
	}


@end
